src/manage_sql_nvts.c

```diff
@@ -1751,9 +1792,10 @@ manage_update_nvt_cache_osp (const gchar *update_socket)
       g_info ("OSP service has newer VT status (version %s) than in database (version %s, %i VTs). Starting update ...",
               scanner_feed_version, db_feed_version, sql_int ("SELECT count (*) FROM nvts;"));
 
-      if (update_nvt_cache_osp (update_socket, db_feed_version,
-                                scanner_feed_version))
-        return -1;
+      ret = update_nvt_cache_osp (update_socket, db_feed_version,
```

Comment:
Just return directly?

Reply:
This was left from an earlier approach where the return code was handled here.
Changed in aaa7768402a4371ccf407ea221647144105817d4